docs/sections/use-case-requirements.html

```diff
@@ -57,14 +70,18 @@ <h4>Credential Exchange</h4>
         products.
       </p>
     </dl>
-    <p>Endpoints of interest:</p>
+    <p>Examples of credential exchange include:</p>
```

Comment:
The real question in all of these is, are these supply chain use cases or are they trace interop use cases. 
I think its the second so the use cases should be much more around 
` I need to send a VP over an authenticated channel` rather than `I need to issue a BOL`

Reply:
I agree on focusing the examples on the "sending" aspect - which is specifically what trace-interop is about. 

To your more philosophical question: surely, the _traceability_-part of "traceability-interop" refers to supply chain. Examples are meant to be relatable. "Issuing and sending a BOL between a carrier and a shipper" is something our target audience can relate to. "VP over an authenticated channel" is esoteric tech-speak IMO - let's focus on that in the normative parts, not the business examples.

Reply:
... I included the technical aspects in the introduction-section.